bvm/ballerina-core/src/main/java/org/ballerinalang/connector/impl/StructImpl.java

```diff
@@ -79,6 +79,13 @@ public long getIntField(String fieldName) {
         return ((BInteger) value.get(fieldName)).intValue();
     }
 
+    public long getDefaultableIntField(String fieldName) {
```

Comment:
Shall we use ternary operator here? WDYT?
return (value.get(fieldName) != null) ? getIntField(fieldName) : 0;